XamarinCommunityToolkit.UnitTests/Converters/ItemSelectedEventArgsConverter_Tests.cs

```diff
@@ -0,0 +1,37 @@
+﻿using XamarinCommunityToolkit.Converters;
+using System.Collections.Generic;
+using System.Globalization;
+using Xamarin.Forms;
+using System;
+using Xunit;
+
+namespace XamarinCommunityToolkit.UnitTests.Converters
+{
+    public class ItemSelectedEventArgsConverter_Tests
+    {
+        public static object expectedValue = 100;
```

Comment:
```suggestion
       static object expectedValue = 100;
```

This doesn't need to be public, right? It's a nit, but just to make sure people aren't finding it with IntelliSense or something :)